about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/83eb6c852a.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./css/about.css">
+  <title>Product Management Workshop</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="index.html" class="logo">PM Global <br />2022</a>
+      <div id="hamburger">
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+        <div class="ham-bar"></div>
+      </div>
+      <div class="menu">
+        <div class="menu-element close">
+          <i class="fa fa-times fa-1x"></i>
+        </div>
+        <ul class="nav-list">
+          <li><a href="index.html" class="menu-element">Home</a></li>
+          <li><a href="about.html" class="menu-element loaded">About</a></li>
+          <li><a href="index.html#main-program" class="menu-element">Program</a></li>
+          <li><a href="index.html#main-program" class="menu-element">Sponsors</a></li>
+          <li><a href="index.html#main-program" class="menu-element">News</a></li>
+          <li><a href="index.html#main-program" class="menu-element" id="campaign">Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="intro">
+      <h1 id="hero-text">
+        Project Managers <br> Global Workshop 2022
+      </h1>
+    </div>
+    <hr class="heading-underline">
+    <p id="keynote">
+      Novustack is beginning the second quarter of 2022 with a fantastic offer for young techies looking to get their 
+      foot in the door of Product Management. We'll be joined by a Harvard and MIT graduate with expertise as a 
+      former Amazon Technical Product Manager for two days in April.
+      <br><br>
+      It promises to be an engaging and action-packed weekend, all for the low price of $25 
+      (N10,000, 181 cedis, 2900 Kenyan shillings).
+      <br><br>
+      Payments made before April 7th attracts a 20% discount.
+    </p>
+    <!-- <p class="event-date">2022.04.19(TUE) ~ 21(THU)</p> -->
+    <p class="reach-us">Please, contact us per Email for any further questions about the PM Global Workshop, 2022!
+      <br>
+      <a href="mailto:info@novustack.com">info@novustack.com</a>
+    </p>
+  </header>
+  <main>
+
+  </main>
+  <section class="past-workshops">
+    <h2>See Past Workshops</h2>
+    
+    <hr class="heading-underline">
+    
+    <p>Take a look at the last two PM Global workshops that took place in Toronto, Canada and California</p>
+    
```

Comment:
- [ ] The middle section with the header, divider, and logo missing from your design. Kindly see what you are missing below and implement it 👇 
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/164912893-0d7214a6-58f8-4427-807f-21faf5dcfb5a.png)